agent360/plugins/bind.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import os
+import plugins
+import json
+
+class Plugin(plugins.BasePlugin):
+    __name__ = 'bind'
+
+    def run(self, config):
+        '''
+        Monitoring bind nameserver
+
+        You must have the bind statistics-channels configured
+        and you must have jq installed for json processing.
+        '''
+        result = os.popen('curl -j http://localhost:8053/json 2>/dev/null | jq ".qtypes * .rcodes * .nsstats"').read()
```

Comment:
Is bind always running on localhost:8053 or should this be configurable by the user?

Reply:
> Is bind always running on localhost:8053 or should this be configurable by the user?

👍 Yeah, the port may depend.  I added a new patch which contains a configuration var for the port.

Reply:
Thanks!